docs/notebooks/model_config.pct.py

```diff
@@ -110,19 +113,79 @@ def build_model(data):
 }
 
 # %% [markdown]
-# Next you simply pass the configuration dictionary to the `optimize` function and `BayesianOptimizer` will sort out which model and optimizer interface needs to be used to build a valid model.
+# Next you simply pass the configuration dictionary to the `optimize` function and `BayesianOptimizer` will sort out which model and optimizer wrapper needs to be used to build a valid model.
 
 # %%
 bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
 result = bo.optimize(2, initial_data, model_config)
 
 
+# %% [markdown]
+# ## Using configuration dictionaries for setting up experiments
+#
+# Another use case is in setting up experiments, where it becomes easier to benchmark Bayesian optimization algorithms. The advantage is that we can easily change the models and set up any argument for them from one condition to another - only object with the experiment specification needs to change (`experiment_conditions` below), while the rest of the code for executing experiments can stay the same. 
```

Comment:
Second sentence is a bit hard to read, can you simplify it? Also what are these "conditions"?

Reply:
Agree. Also, replace - with : or —.